views/logged-in.hbs

```diff
@@ -0,0 +1,3 @@
+    {{!-- Oproepen users van database meegeven? --}}
+    {{> userlist users=users}}
+    {{{body}}}
```

Comment:
Body word al gecalled in de layout, dus dit hoeft hier niet je mist nog wel de {{> header}}